java/gazelle/testdata/attribute_setting/arguments.txt

```diff
@@ -0,0 +1 @@
+-java-annotation-to-attribute=com.example.annotation.FlakyTest=flaky=True
```

Comment:
Is this something that would come from gazelle directive? Can we thus set this in the build file instead?

Reply:
Sadly not currently because of https://github.com/bazelbuild/bazel-gazelle/issues/1313

I'd love to move this to be a directive-driven piece of config, rather than flag-driven, but there's still a decent amount of upstream discussion to have there.